app/src/renderer/components/wallet/LiCoin.vue

```diff
@@ -12,9 +12,17 @@
           {{ amount }}
         </p>
       </div>
+      <!-- disable send on the hub until send is enabled -->
+      <tm-btn
+        v-if="!lastHeader || lastHeader.chain_id === 'cosmoshub-1'"
+        v-tooltip.left="tooltip"
+        value="Send"
+        color="primary"
```

Comment:
oh wait this is not disabled

Reply:
if it's disabled it's not showing the tooltip
But this button doesn't trigger anything

Reply:
should be disabled as button `disabled=disabled`. That's the literal of transfers disabled

Reply:
but then it is confusing as it doesnt state why

Reply:
feel free to change the tooltip for a message on the page or something

Reply:
but can't we disable the button and show the tooltip too ?

Reply:
didn't get it to work

Reply:
hmm ok

Reply:
i believe in chrome disabled buttons don't register the cursor over the button and therefore we can't display tooltips.